### PR TITLE
🧹 refactor: replace `lodash` with `lodash-es`

### PR DESCRIPTION
**Pull request by defqd**
Imported specific methods individually instead of import the whole 'lodash' library

### CODE REVIEW

File: modules/front-end/angular.json
Comment (cosmos-explorer) [inline modules/front-end/angular.json:1]: Hello @defqd , Thank you for your PR.

Have you check the recommended way in this answer? https://stackoverflow.com/a/62589268
With this, we don't need to add **allowedCommonJsDependencies**

Comment (defqd) [reply modules/front-end/angular.json:1]: Hello @cosmos-explorer, I checked. The warning has disappeared.

File: modules/front-end/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "@angular/platform-browser-dynamic": "^14.0.0",
     "@angular/router": "^14.0.0",
     "@antv/g2": "^4.2.7",
+    "lodash-es": "^4.17.21",
     "lodash.differenceby": "^4.8.0",
```
Comment (cosmos-explorer) [inline modules/front-end/package.json]: You can remove 
- lodash.differenceby
- lodash.get
- lodash.intersectionby
- lodash.keyby

Then rerun `npm i`

**Review by cosmos-explorer — APPROVED** [review]

**Comment by cosmos-explorer** [comment]
LGTM! Thank you very much

**Pull request merged**